src/ApiClient.js

```diff
@@ -6,7 +6,7 @@
  */
 
 import superagent from 'superagent';
-import querystring from 'querystring';
+import querystring from 'query-string';
```

Comment:
there are 2 changes in this file that fix support for deep objects. This is the first and it swaps out these two very similar libraries because the original one did not encode the urls correctly

Reply:
This answers my other question, thanks!